src/Domain/Regulation/Enum/VehicleTypeEnum.php

```diff
@@ -37,4 +40,13 @@ public static function exemptedCases(): array
             VehicleTypeEnum::OTHER,
         ];
     }
+
+    public static function critairCases(): array
+    {
+        return [
+            2 => VehicleTypeEnum::CRITAIR_2,
+            3 => VehicleTypeEnum::CRITAIR_3,
+            4 => VehicleTypeEnum::CRITAIR_4_5,
```

Comment:
Pourquoi ne pas séparer 4 et 5 ?

```suggestion
            4 => VehicleTypeEnum::CRITAIR_4,
            5 => VehicleTypeEnm::CRITAIR_5,
```

Reply:
Parce que ça fait partie des critères d'acceptation 
>  (Must have) Les catégories "Crit'Air 4" et "Crit'Air 5" sont regroupées en une seule catégorie "Crit'Air"

Reply:
Alors heureusement que je ne suis pas PO :see_no_evil: C'est une ambigüité de ma part

Je voulais écrire que les actuels boutons "Crit'Air 4" et "Crit'Air 5" sont fusionnés en un seul bouton "Crit'Air"

Mais la maquette fait foi